### PR TITLE
Readme with tips for using python scripts

### PR DESCRIPTION
**Pull request by hannahbergam**
This PR adds a README to the spritesheet generation scripts file. These tips should allow for a smoother process whenever a user has to generate new spritesheets.

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/CSA-623

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: tools/scripts/javalabSpritesheetGenerators/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## For Creating a Single Spritesheet
+
+### Requirements
+
+- Python - An ability to run scripts locally
+- Pillow - IDE Extension
```
Comment (sanchitmalhotra126) [inline tools/scripts/javalabSpritesheetGenerators/README.md]: Could we include URLs for these?

**Review by sanchitmalhotra126 — APPROVED** [review]
Other than potentially adding URLs, looks good to me!

File: tools/scripts/javalabSpritesheetGenerators/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+- TabSave - Or any other site for downloading images from urls in bulk
+
+### Steps
+
+- Within neighborhood sprites google sheet, filter by category (i.e. vehicles, other, etc)
```
Comment (breville) [inline tools/scripts/javalabSpritesheetGenerators/README.md]: Maybe a link to the sheet?  Or is it already well-known internally?

File: tools/scripts/javalabSpritesheetGenerators/README.md (hunk 1 above)
Comment (hannahbergam) [reply tools/scripts/javalabSpritesheetGenerators/README.md]: ✅

File: tools/scripts/javalabSpritesheetGenerators/README.md (hunk 2 above)
Comment (hannahbergam) [reply tools/scripts/javalabSpritesheetGenerators/README.md]: Good call- thanks!

File: tools/scripts/javalabSpritesheetGenerators/README.md (hunk 1 above)
Comment (hannahbergam) [reply tools/scripts/javalabSpritesheetGenerators/README.md]: Good point- added for Pillow and Tab Save!

**Pull request merged**